bin/s2nc.c

```diff
@@ -46,7 +44,7 @@ void usage()
     fprintf(stderr, "  --ciphers [version_string]\n");
     fprintf(stderr, "    Set the cipher preference version string. Defaults to \"default\". See USAGE-GUIDE.md\n");
     fprintf(stderr, "  --enter-fips-mode\n");
-    fprintf(stderr, "    Enter libcrypto's FIPS mode. The linked version of OpenSSL must be built with the FIPS module.\n");
+    fprintf(stderr, "    Throw an error if libcrypto FIPS mode was unsuccessful. The linked version of OpenSSL must be built with the FIPS module.\n");
```

Comment:
Can link more than OpenSSL. Using a generic "libcrypto" is more precise I guess?